.github/workflows/create-jira-ticket.yml

```
@@ -0,0 +1,56 @@
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install md-to-adf
+        if: steps.check_label.outputs.has_label == 'true'
+        run: npm install -g md-to-adf
+
+      - name: Convert issue body to ADF
+        id: convert_adf
+        if: steps.check_label.outputs.has_label == 'true'
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:3:33: Double quote to prevent globbing and word splitting

(shellcheck)